easybuild/easyconfigs/n/NGS_Automated/NGS_Automated-3.4.0-bare.eb

```diff
@@ -0,0 +1,29 @@
+name = 'NGS_Automated'
+version = '3.4.0'
+namelower = name.lower()
+versionsuffix='-bare'
+homepage = 'https://github.com/molgenis/NGS_Automated'
+description = """Various scripts to automate the NGS pipelines implemented in the MOLGENIS Compute framework."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+easyblock = 'Tarball'
+
+#
+# Example URL: https://github.com/molgenis/NGS_Automated/archive/1.0.1.tar.gz
+# 
+source_urls = [('http://github.com/molgenis/%s/archive/' % (name))]
+sources = [('%s.tar.gz' % (version))]
+
+checksums = ['11dfa641d3752e4027f64a091a83e91d8c26e0a03676567725e82f6e2c0be667']
+
+sanity_check_paths = {
+    'files': [
+        'etc/umcg-patho.cfg',
+        'etc/umcg-labgnkbh.cfg',
```

Comment:
Why do we only check for this subset of config files and not for the others including the actual scripts themselves?

Reply:
since those are the new ones, than you actually know that you have the correct release

Reply:
The sanity check for 3.6.3 is already better, but could check a bit more...

Reply:
> since those are the new ones, than you actually know that you have the correct release

When only checking for the new stuff you know that your are not missing the new stuff, but if any of the old stuff that still needs to be installed is missing the sanity check will fail to notice that.